beater/pubsubbeat.go

```diff
@@ -100,6 +105,46 @@ func (bt *Pubsubbeat) Run(b *beat.Beat) error {
 		bt.pubsubClient.Close()
 	}()
 
+	events := make(chan ackableEvent)
+
+	go func() {
+		// batch up documents to prevent contending on lock in client.Publish()
```

Comment:
Minor nit on comment style.
```suggestion
		// Batch up documents to prevent contending on lock in client.Publish().
```